main.tf

```diff
@@ -12,17 +12,27 @@ locals {
   ]
 
   website_endpoints = [
```

Comment:
Can you rename this to `bucket_domain_names`?

Reply:
Yes of course, sorry. It crossed my mind but I forgot to actually rename it.